osu.Desktop/Security/CompatibilityModeChecker.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Linq;
+using Microsoft.Win32;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Graphics;
+using osu.Game.Overlays;
+using osu.Game.Overlays.Notifications;
+
+namespace osu.Desktop.Security
```

Comment:
I figured that this should be in security since it sorta keeps the game in a "secure state" by doing these checks, but I guess a better way is to say that it keeps the game in a stable state by doing these checks?

I could probably make a new folder called "Stability" and put it in there.

Reply:
Just put it in `osu.Game/Windows`.